src/ui/views/PushDetails/PushDetails.jsx

```diff
@@ -310,18 +318,28 @@ export default function Dashboard() {
                         {moment.unix(c.commitTs || c.commitTimestamp).toString()}
                       </TableCell>
                       <TableCell>
-                        <a
-                          href={`https://github.com/${c.committer}`}
-                          rel='noreferrer'
-                          target='_blank'
-                        >
-                          {c.committer}
-                        </a>
+                        {isGitHub && (
+                          <a
+                            href={`https://github.com/${c.committer}`}
+                            rel='noreferrer'
+                            target='_blank'
+                          >
+                            {c.committer}
+                          </a>
+                        )}
+                        {!isGitHub && <span>{c.committer}</span>}
                       </TableCell>
                       <TableCell>
-                        <a href={`https://github.com/${c.author}`} rel='noreferrer' target='_blank'>
-                          {c.author}
-                        </a>
+                        {isGitHub && (
+                          <a
+                            href={`https://github.com/${c.author}`}
+                            rel='noreferrer'
+                            target='_blank'
+                          >
+                            {c.author}
+                          </a>
+                        )}
+                        {!isGitHub && <span>{c.author}</span>}
```

Comment:
@StingRayZA another spot where we could add a gitlab equivalent

Reply:
Could genericise this further and add a `getUserProfileData` that returns the `<span>` or `<a>` accordingly:

```javascript
// earlier in the function definitions
const getUserProfileData = (username, provider, hostname) => {
  let profileData = '';
  const profileUrl = getUserProfileUrl(username, provider, hostname);
  if (profileUrl) {
    profileData = `<a href="${profileUrl}" rel='noreferrer' target='_blank'>${username}</a>`;
  } else {
    profileData = `<span>${username}</span>`;
  }
  return profileData;
}

// Usage in TableCell:
{getUserProfileData(c.author, gitProvider, new URL(repoUrl).hostname)}
```